### PR TITLE
Added 1.4.0 changes

### PR DESCRIPTION
**Pull request by MadelineMurray**
Signed-off-by: Madeline <madeline.murray@consensys.net>

1.4.0 changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+# Changelog
```
Comment (timbeiko) [inline CHANGELOG.md]: "Changelog" is there twice.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+
+### Bug Fixes 
+
+See RC and Beta sections below. 
+
+### Known Bugs 
+
+- Error syncing with mainnet on Besu 1.3.7 node - MerkleTrieException [\#BESU-160](https://jira.hyperledger.org/browse/BESU-160)
+
+Workaround -> Don't enable pruning when syncing to mainnet.
```
Comment (timbeiko) [inline CHANGELOG.md]: @RatanRSur any other suggestion based on your investigation so far?

File: CHANGELOG.md (hunk 1 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Good point - deleted one!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+
+* [Tracing API](https://besu.hyperledger.org/en/latest/Concepts/Transactions/Trace-Types) to obtain detailed information about transaction processing. 
+
+### Bug Fixes 
+
+See RC and Beta sections below. 
+
+### Known Bugs 
+
+- Error syncing with mainnet on Besu 1.3.7 node - MerkleTrieException [\#BESU-160](https://jira.hyperledger.org/browse/BESU-160)
```
Comment (RatanRSur) [inline CHANGELOG.md]: Can we pull more details up into this line? `Error full syncing with pruning`

Also, it doesn't just affect mainnet.

Comment (timbeiko) [reply CHANGELOG.md]: I think that's the name of the Jira ticket. Does it make sense to change L46 to 
> The associated error is `Error full syncing with pruning: Unable to load trie node value for hash`

Comment (MadelineMurray) [reply CHANGELOG.md]: Added more details.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,78 @@
...
+and is planned for inclusion in v1.4.1. 
+
+#### Error full syncing with pruning 
+
+- Error syncing with mainnet on Besu 1.3.7 node - MerkleTrieException [\#BESU-160](https://jira.hyperledger.org/browse/BESU-160)
+The associated error is `Unable to load trie node value for hash` and is caused by the combination of 
+full sync and pruning. 
+
+Workarounds: 
+1. Explicitly disable pruning using `--pruning-enabled=false` when using fast sync.
```
Comment (RatanRSur) [inline CHANGELOG.md]: This should either say full sync or just disable in general. We don't actually know how likely it is to occur for fast sync but the pruning times are spread out enough that it's definitely a much smaller issue than full sync.

**Pull request merged**